agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/LitIndexer.java

```diff
@@ -0,0 +1,221 @@
+package org.alliancegenome.indexer.indexers;
+
+import java.lang.reflect.UndeclaredThrowableException;
+import java.text.SimpleDateFormat;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.concurrent.LinkedBlockingDeque;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.alliancegenome.indexer.config.IndexerConfig;
+import org.alliancegenome.indexer.indexers.curation.LitDocument;
+import org.alliancegenome.core.config.ConfigHelper;
+import org.alliancegenome.indexer.indexers.curation.interfaces.LiteratureElasticSearchInterface;
+
+import si.mazi.rescu.HttpStatusIOException;
+import si.mazi.rescu.RestProxyFactory;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import groovyjarjarpicocli.CommandLine.Help.Ansi.IStyle;
+
+import com.google.gson.JsonObject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LitIndexer extends Indexer {
```

Comment:
As the other indexer use `@Slf4j` annotation at the top of the class. I would suggest switching this over.